src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporter.cs

```diff
@@ -31,7 +31,9 @@ namespace OpenTelemetry.Exporter;
 internal sealed class OtlpLogExporter : BaseExporter<LogRecord>
 {
     private readonly SdkLimitOptions sdkLimitOptions;
+    private readonly ExperimentalOptions experimentalOptions;
```

Comment:
nit: You could probably remove this field and `sdkLimitOptions` I think we're only using them to construct `otlpLogRecordProcessor` in the ctor now.